start.java

```diff
@@ -64,6 +70,7 @@ public static void main(String[] args) {
             System.out.printf("  %s %s%n", "java start.java --https", "# Start Keycloak Environment with https");
             System.out.printf("  %s %s%n", "java start.java --https --database=postgres", "# Start Keycloak Environment with PostgreSQL database");
             System.out.printf("  %s %s%n", "java start.java --https --openldap --database=postgres", "# Start Keycloak Environment with PostgreSQL database and OpenLDAP");
+            System.out.printf("  %s %s%n", "java start.java --extensions=classes", "# Start Keycloak with extensions mounted from classes folder");
```

Comment:
"Start Keycloak with extensions mounted from classes folder. Use --extensions=jar to mount the jar file into the container"